assets/style.css

```diff
@@ -11,13 +11,13 @@ h1 {
   margin-top: 1rem;
   margin-bottom: 3.5rem;
 }
-.card {
```

Comment:
I had to comment this section out in order to make the weather boxes on the second page readable. we might need to add separate css page or add an id to the card.